README.rst

```diff
@@ -60,11 +60,11 @@ You're done!
 Development
 -----------
 
-First, `install conda <http://conda.pydata.org/docs/install/quick.html>`_ and run
+First, `install conda <http://conda.pydata.org/docs/install/quick.html>`_ and run::
 
     $ conda install numpy scipy pytables
 
-After this, SAPPHiRE should work out of the box:
+After this, SAPPHiRE should work out of the box:::
```

Comment:
Two colons do produce one colon, as can be seen in the Installation section.
If you want no colons you can use some code-block tag in reStructuredText.